custom-vu/src/main/kotlin/jces1209/vu/utils/boards/CsvBoardsReader.kt

```diff
@@ -0,0 +1,58 @@
+package jces1209.vu.utils.boards
+
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import java.lang.IllegalStateException
+import java.net.URI
+
+object CsvBoardsReader {
+
+    var boardsList: MutableList<CsvBoard>?
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+    private const val csvPropertyFile = "BoardUsageFrequency.csv"
+
+    init {
+        boardsList = getBoardsFromCsv()
```

Comment:
Let's move the list in BoardsFrequencyManager and count totalWeight only once

Reply:
Ok, got you, will fix this.